content/en/tutorials/changing_domain.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Changing Domains"
+linkTitle: "Changing Domains"
+date: 2021-11-15
+draft: false
+description: >
+  How to change domains for your Opta environment.
+---
+
+Currently, Opta does not support multiple domains (except subdomains) per environment, although that may change
+based on user demand. In order for a user to change the domain tied to their environment, they would need to
+remove their current dns module, and then add a new one after applying. 
+
+Take for example, if we had the current running environment:
+
+```yaml
+name: blah
+org_name: baloney
+providers:
+  aws:
+    region: us-east-1
+    account_id: XXXXXXXXXXXX
+modules:
+  - type: base
+  - type: dns
+    domain: baloney.dev
+  - type: k8s-cluster
+  - type: k8s-base
+```
+
+Supposed we wished to change the domain to "otherdomain.dev". First we would remove the dns module:
+
+```yaml
+name: blah
+org_name: baloney
+providers:
+  aws:
+    region: us-east-1
+    account_id: XXXXXXXXXXXX
+modules:
+  - type: base
+  - type: k8s-cluster
+  - type: k8s-base
+```
```

Comment:
What happens between these two steps? The site will basically be offline... right? We should call that out.

Reply:
great catch, just added